### PR TITLE
docs: updates README.md with Getting Started and design info

### PR DESCRIPTION
**Pull request by jpower432**
Updates docs to use the same structure as `oscal-component-definitions`. This replaces the manual tutorial (which is out of date) with a tutorial about how to interact with the repostiroy content through GitHub and `trestle-bot` actions. In depth approach information (like https://github.com/RedHatProductSecurity/oscal-component-definitions/blob/main/docs/design.md) is also added.

[Testing](https://github.com/jpower432/oscal-profiles/pull/24) CI naming changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,71 +4,64 @@ OSCAL Custom Profiles for testing with FedRAMP REV5 HIGH baseline profile.
...
-
-To complete work from a fork, local automation is available. To see the available make targets, use `make help`. For information on how to edit the content in this repository, see the [tutorial](./docs/tutorial.md).
+## Getting Started
+
+### Update Existing Content
+#### Overview
+The workflow to update content consists of the following steps:
+
+1. From a user's branch, modify and commit a markdown or json file within the workspace. (Most updates will be made to markdown files.)
+2. A user submits a pull request to add their changes to the workspace's master branch.
```
Comment (gvauter) [inline README.md:14]: Suggest replacing `master` with `main` to align with step 4.

File: docs/design.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Workflows
+
+#### Create
+
+Associated Workflows
+- [create-new.yml](../.github/workflows/create-new.yml)
+
+The `create-new` workflow is triggered manually by going to the Action tab. This creates a new OSCAL profile in the trestle workspace. A new branch is created and a pull request is opened.
+
+#### AutoFix
```
Comment (gvauter) [inline docs/design.md:14]: Would it ease understanding if this was titled "AutoSync" to align with the trestle-bot terminology?

**Review by gvauter — CHANGES_REQUESTED** [review]

**Comment by jpower432** [comment]
@gvauter Please take another look. Due to changes in the CI files, this might require some testing on my fork as well.

**Review by gvauter — APPROVED** [review]
LGTM

**Pull request merged**